02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,26 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>capital factory fitness</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <img src="http://s1.evcdn.com/images/block/I0-001/028/682/924-4.png_/capital-factory-newco-atx-24.png"/>
+<div style= "background-color:grey; color: white; border-radius: 4px;"">
+  <h1>Capital Factory Fitness</h1>
+</div>
+  <h3>Keep Austin Fit</h3>
+    <p>paragraph</p>
+  <h3>Programs offered</h3>
+    <table>
+      <tr>
+        <th><img src="https://pbs.twimg.com/profile_images/638327600486719488/Z-1qM3qs_400x400.jpg"/></th>
+          <td>yoga</td>
+        <th><img src="http://static.wixstatic.com/media/f0df91_41c602d5d2154bfb94bf248939a18d01.jpg_256"/></th>
+          <td>cycling</td>
+        <th><img src="https://pbs.twimg.com/profile_images/595624096861331457/cYqHxMGn.jpg"/></th>
+          <td>crossfit</td>
+      </tr>
```

Comment:
Good work on this code. I'd like to see at least 4 sentences where the paragraph is and please include the contact information described in the directions. Overall, great effort!